packages/core/src/generator/index.ts

```diff
@@ -45,7 +45,10 @@ export class UnoGenerator {
 
     for (const extractor of this.config.extractors) {
       const result = await extractor.extract(context)
-      result?.forEach(t => set.add(t))
+      if (result) {
+        for (const token of result)
+          set.add(token)
+      }
```

Comment:
It seems to be no different from the original code, and the original number of lines is less.